packages/react-examples/.storybook/preview.js

```diff
@@ -5,68 +5,69 @@ import 'cypress-storybook/react';
 import { withPerformance } from 'storybook-addon-performance';
 import { withKeytipLayer, withStrictMode } from '@fluentui/storybook';
 
-/**
- * This placeholder will be replaced with the actual package name by custom webpack loader `./preview-loader.js`
- * @type {string}
- */
-const packageNamePlaceholder = 'PACKAGE_NAME';
-
 initializeIcons();
 
+// This API is deprecated (in favor of `export const decorators = []`) but the new way appears not
```

Comment:
Comment because I tried replacing these with the new way and discovered it doesn't work right now. 😞  My guess is it has to do with using `configure()` (which is also deprecated) but I'm not sure.